src/lib/endpoints.js

```diff
@@ -1,6 +1,15 @@
-import { httpGet } from "./dataAccess";
+import { httpGet, httpPost } from "./dataAccess";
 
 export const getAllScenarios = async () => {
     const response = await httpGet("/api/admin/v1/scenarios");
     return response.data.scenarios;
 };
+
+export const postScenario = async ({ name, description, friendly_name }) => {
+    const response = await httpPost("/api/admin/v1/scenario", {
```

Comment:
I think we should move `/api/admin/v1/` to a env variable so that if we change to v2 in the future, it's an easy change.

Can you also retroactively make this change to the `getAllScenarios`?